docs/csharp/linq/standard-query-operators/index.md

```diff
@@ -27,6 +27,8 @@ Where possible, the queries in this section use a sequence of words or numbers a
 
 Each `Student` has a grade level, a primary department, and a series of scores. A `Teacher` also has a `City` property that identifies the campus where the teacher holds classes. A `Department` has a name, and a reference to a `Teacher` who serves as the department head.
 
+You can find the data set in the [source repo](https://github.com/dotnet/docs/blob/main/docs/csharp/linq/standard-query-operators/snippets/standard-query-operators/DataSources.cs#L41).
```

Comment:
I didn't update this using the include page because the description is different compared to the sub-pages.